### PR TITLE
Fixed footer alignment

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**
Fixed footer container, elements styles and rewrited some code. [882](https://github.com/ita-social-projects/GreenCity/issues/882)

1024px:
image

768px:
image

576px:
image

320px:
image

### CODE REVIEW

File: src/app/component/layout/components/footer/footer.component.scss [hunk 1]
```diff
@@ -1,27 +1,22 @@
 @import url('https://fonts.googleapis.com/css?family=Montserrat|Open+Sans|Roboto&display=swap');
 
-$generalFont : OpenSans, sans-serif;
+$generalFont : 'Open Sans';
 
 .main-background-image {
-  background: url('/assets/img/icon/footer/background-image-footer.svg');
-  background-repeat: no-repeat;
-  background-position: center;
+  background: url('src/assets/img/icon/footer/background-image-footer.svg') no-repeat center;
```
Comment (mplesha) [inline src/app/component/layout/components/footer/footer.component.scss:6]: why did you change here?

Comment (RomanDeveloperAcc) [reply src/app/component/layout/components/footer/footer.component.scss:6]: > why did you change here?

Is it better to leave 3 styles, if you can write them in one? It is something similar to margin etc: instead of writing margin-top, margin-bottom... you may write only margin

Comment (mplesha) [reply src/app/component/layout/components/footer/footer.component.scss:6]: it's not about this, but about:
```css
background: url('src/assets/img/icon/footer/background-image-footer.svg');
```
vs
```css
background: url('/assets/img/icon/footer/background-image-footer.svg');
```
